lib/Controller.h

```diff
@@ -165,6 +165,13 @@ const ubyte CT_X_AXIS = 1, // AXIS constants for ctAxis
     CT_Y_AXIS = 2, CT_Z_AXIS = 3, CT_R_AXIS = 4, CT_U_AXIS = 5, CT_V_AXIS = 6,
             CT_NUM_AXES = 6; // number of axes
 
+#define makeword(_h, _l) (((_h) << 16) + ((_l) & 0xffff))
+#define hiword(_v) ((_v) >> 16)
+#define loword(_v) ((_v) & 0x0000ffff)
+#define makeshort(_h, _l) (((_h) << 8) + ((_l) & 0x00ff))
+#define hibyte(_w) ((_w) >> 8)
+#define lobyte(_w) ((_w) & 0x00ff)
+
```

Comment:
Why these macros moved to particular windows controller.h?

Reply:
TL;DR: this is best location for them to be.

Full explanation:
`Controller.h` is included by both `lnxcontroller/lnxcontroller.cpp` and `win32/WinController.cpp`

Referenced in `Controller.h`, `lnxcontroller/lnxcontroller.cpp`, and `win32/WinController.cpp`
* `makeword`

Referenced in `Controller.h` and `winjoy.cpp`
* `loword`

Only referenced in `Controller.h`
* `makeshort`
* `hiword`
* `hibyte`
* `lobyte`

`Controller.h` is therefore the logical location for these macros. A copy of `loword` was placed in `winjoy.cpp`.

It makes no sense for dozens of files to have these macros.

Reply:
I'm fine with that